src/foam/nanos/crunch/extra/ReviewCapability.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.crunch.extra',
+  name: 'ReviewCapability',
+  extends: 'foam.nanos.crunch.Capability',
+  documentation: `
+    This capability displays a portal to another capability as well
+    as its own data. This is useful for implementing a review process.
+  `,
+
+  properties: [
+    {
+      class: 'Object',
+      name: 'wizardlet',
+      documentation: `
+        Defines a wizardlet used when displaying this capability on related client crunch wizards.
+      `,
+      factory: function() {
+        return foam.nanos.crunch.ui.ReviewCapabilityWizardlet.create({}, this);
```

Comment:
Better to require: this class then do this.ReviewCapabilityWizardlet() instead. Helps the build follow dependencies better.